File: subset4/if.pl
#!/usr/bin/perl -w

$number = <STDIN>;
if ($number >= 0 || $number <= 10) {
    if ($number % 2 == 0) {
        print "Even\n";
    } else {
        print "Odd\n";
    }
}
print "Bye\n";

File: subset4/if.py
#!/usr/local/bin/python -u
import sys

number = sys.stdin.readline()
number  = float(number )
if number >= 0 or number <= 10:
    if number % 2 == 0:
        print("Even")
    else:
        print("Odd")
print("Bye")


